framework/src/modules/interoperability/schema.ts

```diff
@@ -11,3 +11,138 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+
+export const chainAccountSchema = {
+	$id: 'modules/interoperability/chainAccount',
+	type: 'object',
+	required: [
+		'inbox',
+		'outbox',
+		'networkID',
+		'lastCertifiedStateRoot',
+		'lastCertifiedTimestamp',
+		'lastCertifiedHeight',
+		'partnerChainOutboxRoot',
+		'partnerChainOutboxSize',
+		'partnerChainInboxSize',
+		'name',
+		'status',
+		'activeValidators',
+		'certificateThreshold',
+	],
+	properties: {
+		inbox: {
+			type: 'object',
+			fieldNumber: 1,
+			required: ['appendPath', 'size', 'root'],
+			properties: {
+				appendPath: {
+					type: 'array',
+					items: {
+						dataType: 'bytes',
+					},
+					fieldNumber: 1,
+				},
+				size: {
+					dataType: 'uint64',
+					fieldNumber: 2,
+				},
+				root: {
+					dataType: 'bytes',
+					fieldNumber: 3,
+				},
+			},
+		},
+		outbox: {
+			type: 'object',
+			fieldNumber: 2,
+			required: ['appendPath', 'size', 'root'],
+			properties: {
+				appendPath: {
+					type: 'array',
+					items: {
+						dataType: 'bytes',
+					},
+					fieldNumber: 1,
+				},
+				size: {
+					dataType: 'uint64',
+					fieldNumber: 2,
+				},
+				root: {
+					dataType: 'bytes',
+					fieldNumber: 3,
+				},
+			},
+		},
+		networkID: {
+			dataType: 'bytes',
+			fieldNumber: 3,
+		},
+		lastCertifiedStateRoot: {
+			dataType: 'bytes',
+			fieldNumber: 4,
+		},
+		lastCertifiedTimestamp: {
+			dataType: 'uint32',
+			fieldNumber: 5,
+		},
+		lastCertifiedHeight: {
+			dataType: 'uint32',
+			fieldNumber: 6,
+		},
+		partnerChainOutboxRoot: {
+			dataType: 'bytes',
+			fieldNumber: 7,
+		},
+		partnerChainOutboxSize: {
+			dataType: 'uint64',
+			fieldNumber: 8,
+		},
+		partnerChainInboxSize: {
+			dataType: 'uint64',
+			fieldNumber: 9,
+		},
+		name: {
+			dataType: 'string',
+			fieldNumber: 10,
+		},
+		status: {
+			dataType: 'uint32',
+			fieldNumber: 11,
+		},
+		activeValidators: {
+			type: 'array',
+			fieldNumber: 12,
+			items: {
+				type: 'object',
+				required: ['blsKey', 'bftWeight'],
+				properties: {
+					blsKey: {
+						dataType: 'bytes',
+						fieldNumber: 1,
+					},
+					bftWeight: {
+						dataType: 'uint64',
+						fieldNumber: 2,
+					},
+				},
+			},
+		},
+		certificateThreshold: {
+			dataType: 'uint64',
+			fieldNumber: 13,
+		},
+	},
+};
+
+export const terminatedChain = {
```

Comment:
```suggestion
export const terminatedChainSchema = {
```

Reply:
I think it should rather be `terminatedStateSchema` as per the [PR](https://github.com/LiskHQ/lips/pull/131/files#diff-a3ced5b919eed85370156537064b46119c1e19204e1d348b91f757a0f5ca8d8bL422)